### ISSUE TITLE
[BUG] Set-PnPListItem -Batch fails on update of Managed Metadata Column

### ISSUE THREAD

**Issue by PHubaut** [pnp/powershell#340]
### Expected behavior 
Set-PnPListItem allows batching as per the documentation.
Any Set-PnPListItem that works without "-Batch" is expected to also work with "-Batch".

### Actual behavior 
Set-PnPListItem used to update a multivalued managed metadata column works outside a batch but fails when added to a batch. The Set-PnPListItem call ends with the following error:
```
Set-PnPListItem : Specified argument was out of the range of valid values.
Parameter name: lcid
At xxx...xxx.ps1:nn char:n
+ Set-PnPListItem -List "BI" -Identity 45934 -Values @{
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : WriteError: (:) [Set-PnPListItem], ServerException
    + FullyQualifiedErrorId : EXCEPTION,PnP.PowerShell.Commands.Lists.SetListItem
```

### Steps to reproduce behavior
Assuming a library "BI", document 45934 and two terms "f5d4baae-6d12-42bf-b3f1-76f06ef0f037", "bca7c6ab-a948-4cd1-b1b6-38d9001b47bc" from the terms set linked to a multivalued managed metadata column named "Tags". The following code fails:
```
$Batch = New-PnPBatch
Set-PnPListItem -List "BI" -Identity 45934 -Values @{
    "Tags" = "f5d4baae-6d12-42bf-b3f1-76f06ef0f037", "bca7c6ab-a948-4cd1-b1b6-38d9001b47bc"
} -Batch $Batch
Invoke-PnPBatch -Batch $Batch
```
Although this one works:
```
Set-PnPListItem -List "BI" -Identity 45934 -Values @{
    "Tags" = "f5d4baae-6d12-42bf-b3f1-76f06ef0f037", "bca7c6ab-a948-4cd1-b1b6-38d9001b47bc"
}
```
### What is the version of the Cmdlet module you are running?
```
Version          Name                                   Repository
-------          ----                                   ----------
1.3.14-nightly   PnP.PowerShell                         PSGallery
```
On PSVersion
```
Major  Minor  Build  Revision
-----  -----  -----  --------
5      1      17763  1490
```

### Which operating system/environment are you running PnP PowerShell on?
- [X] Windows

**Comment by erwinvanhunen** [comment]
I just tried exactly what you tried, e.g. I created a column called Tags on a new list, managed metadata, marked is as a multi-value column.

I try in two ways to add two values to the list item, and that succeeded twice...

![image](https://user-images.githubusercontent.com/7666381/109180363-1451d980-778b-11eb-97ac-8a520060cc73.png)

![image](https://user-images.githubusercontent.com/7666381/109180546-4400e180-778b-11eb-86f5-8efa3ee3551d.png)

What is the error message you are receiving?

**Comment by PHubaut** [comment]
As the document library  I am working on is more complex, I simplified and did the same you did.
I created "ItemTest" list with Tags column pointing to the same terms set.

I noticed you used Add-PnPListItem (I was referring to Set-PnPListItem) but it is the same behavior -

Here is what I get - 1st call (without batch is ok); 2nd call with batch, I get a `Specified argument was out of the range of valid values. Parameter name: lcid` error:
![image](https://user-images.githubusercontent.com/22426791/109184263-1ae25000-778f-11eb-8d52-3474af06f787.png)

**Comment by erwinvanhunen** [comment]
I have a guess where it goes wrong: the current locale you're running in, is that different from the actual term store? E.g. you're running in English (1033) but the termstore does not have labels available for that locale? In order for us to add the value to the field, we need to retrieve a label. We default to the label for the current culture you're using in the current session.

I'm looking at a solution now.

**Comment by erwinvanhunen** [comment]
I just merged a change, which my guess is will solve your problem. Please check it out if you can, this will be available in tomorrows nightly build: 1.3.16-nightly.

**Comment by PHubaut** [comment]
I'll certainly do - thanks a lot for this - I will report here the results.

**Comment by PHubaut** [comment]
> I just merged a change, which my guess is will solve your problem. Please check it out if you can, this will be available in tomorrows nightly build: 1.3.16-nightly.

Tested on 1.3.15-nightly, works perfectly - thanks a lot.
